playwright-tests/tests/discussions.spec.js

```diff
@@ -142,3 +154,156 @@ test.describe("Wallet is connected", () => {
     expect(await transactionConfirmationModal.isVisible()).toBeFalsy();
   });
 });
+
+test.describe("Don't ask again enabled", () => {
+  test.use({
+    storageState:
+      "playwright-tests/storage-states/wallet-connected-with-devhub-access-key.json",
+  });
+
+  const communitydiscussionaccount =
+    "discussions.webassemblymusic.community.devhub.near";
+
+  test("should create a discussion", async ({ page }) => {
+    await modifySocialNearGetRPCResponsesInsteadOfGettingWidgetsFromBOSLoader(
+      page
+    );
+
+    let discussion_created = false;
+    await mockSocialIndexResponses(page, ({ requestPostData, json }) => {
+      if (
+        requestPostData.action === "repost" &&
+        requestPostData.options?.accountId?.[0] === communitydiscussionaccount
+      ) {
+        console.log("Returning discussions from index", discussion_created);
```

Comment:
do we need logs inside test?

Reply:
No, not mandatory. The reason for logging here was to track where we are intercepting calls to the social index API. I can remove it if it's too annoying.

Reply:
No it’s fine 👍